From 239c493c4951ad56fb246272cda75041f792979e Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Fri, 12 Jul 2019 09:52:14 +0300
Subject: [PATCH 32/33] ucm-imx8m-mini: dts: Fix lvds device tree

Fix lvds device tree.
Disable panel in order to allow lvds port get active.

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 arch/arm64/boot/dts/compulab/ucm-imx8m-mini-lvds.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/compulab/ucm-imx8m-mini-lvds.dts b/arch/arm64/boot/dts/compulab/ucm-imx8m-mini-lvds.dts
index 4e749ef..e9511ef 100644
--- a/arch/arm64/boot/dts/compulab/ucm-imx8m-mini-lvds.dts
+++ b/arch/arm64/boot/dts/compulab/ucm-imx8m-mini-lvds.dts
@@ -35,6 +35,10 @@
     status = "okay";
 };
 
+&panel {
+    status = "disabled";
+};
+
 &mipi_dsi {
     status = "okay";
     port@1 {
-- 
1.9.1

